### PR TITLE
Adds help message for `config`

### PR DESCRIPTION
**Pull request by JJ**
But it would be probably better to generate them automatically from the `command` file. 
Ruby version also bumped down to stable version.

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-2.2.0
+2.1.5
```
Comment (rejeep) [inline .ruby-version]: Can you please remove this.

Comment (JJ) [inline .ruby-version]: Done

**Comment by rejeep** [comment]
Merged, thanks! :+1:

**Comment by JJ** [comment]
Thanks for your good work :-)

**Pull request merged**